app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
```

Comment:
change import order, first build-in, after your imports with an empty line between them